.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Windows PowerShell
+    url: https://windowsserver.uservoice.com/forums/301869-powershell
+    about: Windows PowerShell issues or suggestions.
+  - name: Support
+    url: https://github.com/PowerShell/PowerShell/blob/master/.github/SUPPORT.md
+    about: PowerShell Support Questions/Help
+  - name: Documentation Issue
+    url: https://github.com/MicrosoftDocs/PowerShell-Docs
+    about: Please open issues on documentation for PowerShell here.
+  - name: Security Issue
+    url: https://github.com/PowerShell/PowerShell/blob/master/.github/SECURITY.md
+    about: Reporting security issues for PowerShell.
```

Comment:
We could add
https://github.com/PowerShell/PowerShellModuleCoverage
https://github.com/PowerShell/PSReadLine
https://github.com/Azure/azure-powershell

Reply:
I'm going to leave those off for now as I don't see many issues coming in that necessitate adding them.  Don't want to have a huge menu that makes it more confusing.